interfaces/badger/IBadgerRegistryV2.sol

```diff
@@ -61,6 +61,9 @@ interface IBadgerRegistryV2 {
   function set(string memory key, address at) external;
   function deleteKey(string memory key) external;
   function deleteKeys(string[] memory _keys) external;
+	function governance() external view returns (address);
```

Comment:
Not sure why GH show it indented. Doesn't look like this locally...

Reply:
Probably because of the tabs instead of spaces